source/scss/_library/_module.header.scss

```diff
@@ -4,6 +4,187 @@
 
 .c-main-header {
   text-align: center;
-  background-color: $c-secondary;
-  padding-top: $pad;
+  padding-top: 5px;
+
+  &__ad-unit {
+    position: relative;
+    z-index: 100;
+    background-color: $c-white;
+  }
+
+  &__inner {
+    background-color: $c-heading-bgcolor;
+    padding: $space-half $space-half $space-half 50px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    position: relative;
+    height: 70px;
+
+    @include media('>xsmall') {
+      padding: $space-half $space $space-half 70px;
+    }
+  }
+
+  &__branding {
+    width: 120px;
+    transform: translateY(5px);
```

Comment:
don't love this. seems kind of arbitrary. Have you considered making the svg `display:block;` in order to collapse the whitespace instead?

Reply:
I can try this. I was trying to get it to match the comps exactly. It looks like it's just _slightly_ off vertical center.

Reply:
heh well my preference is to defer to making the CSS portable even if it's at the cost of pixel perfection 😈

Reply:
This is probably one of those things that should be discussed when we have the conversation about how these sites should be themed.